calculator/calculator/calculatorButton/ButtonList1to+.swift

```diff
@@ -0,0 +1,237 @@
+//
+//  ButtonList1to+.swift
+//  calculator
+//
+//  Created by Byeon jinha on 2022/05/20.
+//
+
+import SwiftUI
+
+struct ButtonList1to_: View {
+    @EnvironmentObject var CalculatorDataSheet : CalculatorDataSheet
+    var body: some View {
```

Comment:
```suggestion
    @EnvironmentObject var CalculatorDataSheet : CalculatorDataSheet

    var body: some View {
```

연관이 적은 property, method 들 사이에 개행이 있으면 더 읽기 쉽답니다